src/mixins/translatable.ts

```diff
@@ -25,34 +27,25 @@ export default {
   },
 
   methods: {
+    calcDimensions () {
+      this.parallaxDist = this.imgHeight - this.height
+      this.windowHeight = window.innerHeight
+      this.windowBottom = window.pageYOffset + this.windowHeight
+    },
     listeners () {
       window.addEventListener('scroll', this.translate, false)
       window.addEventListener('resize', this.translate, false)
     },
-
+    objHeight: (): number => 0,
```

Comment:
This might be going a bit overboard but https://github.com/vuetifyjs/vuetify/blob/c1404ab099748f83755cc954a994c77564529d69/src/mixins/button-group.ts#L34-L38